Athenaakrami patch 1 (#290)
NeuromatchAcademy/course-content: projects/README.md

>   The NMA projects are conducted over 14 days, beginning on the morning of the ‘second’ curriculum day. Note that the the schedule of NMA is complex, due to the fact that we operate worldwide and have to accommodate three ‘zones’ and many timezones. The timezones are grouped into A, B and C groupings. This is the structure of the NMA times (note: times are in GMT+1):

Reviewer: ```suggestion
  The NMA projects are conducted over 14 days, beginning on the morning of the ‘second’ curriculum day (W1D2).
```

> ![Calendar](Caltz.png)

Reviewer: ```suggestion
```

There are fourteen Project days, falling between the fifteen content days. Daily project time can be:

- TimeZones 1A, 2A, 3A: Project Day 2 is the morning of course Day 3
- TimeZones 1B, 2B, 3B: Project Day 2 is partly the afternoon of Course Day 2, and partly the morning of Course Day 3.
- TimeZones 1C, 2C, 3C: Project Day 2 is the afternoon of Course Day 2.

> ## Week One

Reviewer: ```suggestion
```

In weeks two and three, you will oversee your students projects. Working with mentors matched to the projects, your role is to answer questions—especially coding or technical questions—and guide the projects. You should also ensure the projects are on track by estimating if they will be done by the middle of week three. Day four of week three should be working on a presentation of the project. In the last day, projects will be presented.

> ### **Week One**

Reviewer: ```suggestion
## Week One
```

Week one begins on the morning of curriculum day two (project day one). During the first day, your role is to encourage students to share their scientific background, interests and goals. It’s a icebreaking session, with emphasis on scientific backgrounds, interests and big questions they might have in neuroscience that would like to explore in future.  For project day two, you will help going through the videos (Project Intro, Curriculum overview, Dataset Intro, and Brainstorm Example). You will control the Zoom interactions and support the students in this process (put them in breakout rooms to discuss the videos). Day three or four, a mentor will join to help with the refinement process: help students to converge on a few projects topics, form groups and work on a project proposal that includes a question, data set and planned techniques to apply to answer the question. We are suggesting you to use the [Miro](https://miro.com/) collaborative board for brainstorming. The project day for the students are:

> * **Project Day 1** — General discussion 

Reviewer: ```suggestion
```

>    * Extended ice-breaker. Encourage students to get to know each other’s scientific interests (this is very important later to form Project Groups, based on shared interests)

Reviewer: ```suggestion
```

>    

Reviewer: ```suggestion
```

 - C zones — Late — 4h after content

> ## **Your schedule**

Reviewer: ```suggestion
```

> There are fourteen Project days, falling between the fifteen content days. Your project time can be:

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

> - TimeZones 1A, 2A, 3A: Project Day 2 is the morning of course Day 3

Reviewer: ```suggestion
```

> - TimeZones 1B, 2B, 3B: Project Day 2 is partly the afternoon of Course Day 2, and partly the morning of Course Day 3.

Reviewer: ```suggestion
```

> - TimeZones 1C, 2C, 3C: Project Day 2 is the afternoon of Course Day 2.

Reviewer: ```suggestion
```
